src/App.tsx

```diff
@@ -1,10 +1,11 @@
-import React from 'react';
-import './App.css';
+import React from "react";
```

Comment:
This line is not necessary in modern React, so go ahead and remove it `import React from react`

Reply:
gotcha